modules/special-rules/pages/pulsar-x.adoc

```diff
@@ -0,0 +1,19 @@
+= Pulsar (X)
+
+This weapon is unique to the Eldar race, and it demonstrates their mastery of laser weaponry.
+
+The weapon shoots in rapid but powerful bursts, guided by advanced targeting technologies.
+As a result, the weapon can destroy multiple targets in a single salvo.
+
+---
+
+When you shoot with a {pulsar-x}:
+
+. Determine the number of shots, based on the value of `(X)` that we specify — for example, _`(D3+1)`_ or _`(D6)`_.
+. Resolve that many {macro-weapon} shots against the target detachment.
+
+IMPORTANT: Remember that a {pulsar-x} is a single super-heavy weapon.
+It does *not* count as multiple super-heavy weapons for placement of Blast markers, suppression, contribution to firefights, or any other game purpose.
```

Comment:
To tie in with my other suggestions:
```suggestion
IMPORTANT: Remember, a salvo of {macro-weapon} shots from a {pulsar} weapon does *not* count as multiple super-heavy weapons for placement of Blast markers, suppression, contribution to firefights, or any other game purpose.
```

These several separate but related suggestions I've made this evening are getting messy — so I think I'll apply them, so we can see how they really look together. Then we can discuss/tweak further as necessary.